dev/com.ibm.websphere.appserver.features/visibility/public/mpRestClient-1.4/com.ibm.websphere.appserver.mpRestClient-1.4.feature

```diff
@@ -19,7 +19,7 @@ Subsystem-Name: MicroProfile Rest Client 1.4
  com.ibm.websphere.appserver.jaxrsClient-2.1, \
  com.ibm.websphere.appserver.jsonp-1.1, \
  com.ibm.websphere.appserver.org.reactivestreams.reactive-streams-1.0, \
- com.ibm.websphere.appserver.mpConfig-1.4; ibm.tolerates:=1.3
+ com.ibm.websphere.appserver.mpConfig-1.4; ibm.tolerates:="1.3,2.0"
```

Comment:
Please remove this tolerates. MicroProfile 4.0 features are not compatible with earlier versions.